apps/full-site-editing/full-site-editing-plugin/jetpack-timeline/blocks/src/timeline-item.js

```diff
@@ -27,29 +34,62 @@ export function registerTimelineItemBlock() {
 				borderColor: attributes.background,
 			};
 
+			const toggleAlignment = ( alignment ) => {
+				const newAlignment = alignment === attributes.alignment ? 'auto' : alignment;
+				setAttributes( { alignment: newAlignment } );
+			};
+
+			const classes = classnames( 'wp-block-jetpack-timeline-item', {
+				'is-left': attributes.alignment === 'left',
+				'is-right': attributes.alignment === 'right',
+			} );
+
 			return (
-				<li style={ style }>
-					<InspectorControls>
-						<PanelColorSettings
-							title={ __( 'Color Settings', 'full-site-editing' ) }
-							colorSettings={ [
-								{
-									value: attributes.background,
-									onChange: ( background ) => setAttributes( { background } ),
-									label: __( 'Background Color', 'full-site-editing' ),
-								},
-							] }
-						/>
-					</InspectorControls>
-					<div className="timeline-item">
-						<div className="timeline-item__bubble" style={ bubbleStyle } />
-						<div className="timeline-item__dot" style={ style } />
-						<InnerBlocks template={ [ [ 'core/heading' ] ] } />
-					</div>
-				</li>
+				<>
+					<BlockControls>
+						<ToolbarGroup>
+							<ToolbarButton
+								onClick={ () => toggleAlignment( 'left' ) }
+								isActive={ attributes.alignment === 'left' }
+								icon={ positionLeft }
+								title={ __( 'Left', 'full-site-editing' ) }
+							/>
+							<ToolbarButton
+								onClick={ () => toggleAlignment( 'right' ) }
+								isActive={ attributes.alignment === 'right' }
+								icon={ positionRight }
+								title={ __( 'Right', 'full-site-editing' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **162** times:
`translate( 'Right' )` *ES Score: 10*